docs/src/estimators_cheatsheet.md

```diff
@@ -0,0 +1,298 @@
+Y &= f_Y(T, W, U_Y)
+\end{aligned}
+```
+
+This is certainly the most common scenario in causal inference where ``Y`` is an outcome of interest, ``T`` a set of treatment variables and ``W`` a set of confounding variables. We are generally interested in the effect of ``T`` on ``Y``. In this document we will consider the counterfactual mean as a workhorse. We will show that the estimators for the Average Treatment Effect and Average Interaction Effect can easily be derived from it. Under usual conditions, the counterfactual mean identifies to the following statistical estimand:
+
+```math
+CM_t(\mathbb{P}) = \Psi_t(\mathbb{P}) = \mathbb{E}_{\mathbb{P}}[\mathbb{E}_{\mathbb{P}}[Y | W, T = t]] = \int \mathbb{E}_{\mathbb{P}}[Y | W, T = t] d\mathbb{P}(w)
+```
+
+From this definition, we can see that ``\Psi_t`` depends on ``\mathbb{P}`` only through two relevant factors:
+
+```math
+\begin{aligned}
+Q_Y(W, T) &= \mathbb{E}_{\mathbb{P}}[Y | W, T] \\
+Q_W(W) &= \mathbb{P}(W)
+\end{aligned}
+```
+
+So that ``\Psi(\mathbb{P}) = \Psi(Q_Y, Q_W)`` (the ``t`` subscript is dropped as it is unambiguous). This makes explicit that we don't need to estimate ``\mathbb{P}`` but only the relevant factors ``(Q_Y, Q_W)`` to obtain a plugin estimate of ``\Psi(\mathbb{P})``. Finally, it will be useful to define an additional factor:
+
+```math
+g(W, T) = \mathbb{P}(T | W) 
+```
+
+Since the gradient of ``\Psi``:
+
+```math
+\phi_{CM_{t}, \mathbb{P}}(W, T, Y) = \frac{\mathbb{1}(T = t)}{g(W, t)}(Y − Q_Y(W, t)) + Q_Y(W, t) − \Psi(\mathbb{P}) 
+```
+
+which is the foundation of semi-parametric estimation, depends on this so-called nuisance parameter.
+
+### Average Treatment Effect and Average Interaction Effect?
+
+They are simple linear combinations of counterfactual means and so is their gradients.
+
+#### Average Treatment Effect
+
+In all generality, for two values of a categorical treatment variable ``T: t_{control} \rightarrow t_{case}``, the Average Treatment Effect (ATE) is defined by:
+
+```math
+ATE_{t_{case}, t_{control}}(\mathbb{P}) = CM_{t_{case}}(\mathbb{P}) - CM_{t_{control}}(\mathbb{P}) 
+```
+
+And it's associated gradient is:
+
+```math
+\begin{aligned}
+\phi_{ATE}(W, T, Y) &= \phi_{CM_{t_{case}}}(W, T, Y) - \phi_{CM_{t_{control}}}(W, T, Y) \\
+&=  H(W, T)(Y − Q_Y(W, T)) + C(W) − ATE_{t_{case}, t_{control}}(\mathbb{P})
+\end{aligned}
+```
+
+with:
+
+```math
+\begin{aligned}
+  \begin{cases}
+    H(W, T) &= \frac{(-\mathbb{1}(T \in (t_{case}, t_{control})))^{T=t_{control}}}{g(W, T)} \\
+    C(W) &= (Q_Y(W, t_{case}) - Q_Y(W, t_{control}))
+  \end{cases}
+\end{aligned}
+```
+
+``H`` is also known as the clever covariate in the Targeted Learning literature (see later).
+
+#### Average Interaction Effect
+
+For simplicity, we only consider two treatments ``T = (T_1, T_2)`` such that: ``T_1: t_{1,control} \rightarrow t_{1,case}`` and ``T_2: t_{2,control} \rightarrow t_{2,case}``. The Average Interaction Effect (AIE) of ``(T_1, T_2)`` is defined as:
```

Comment:
Suggestion: " ... such that ``T_1: t_{1,control} \rightarrow t_{1,case}`` and ``T_2: t_{2,control} \rightarrow t_{2,case}``. " (I would avoid double colons closely the one after the other.)

Reply:
Sorry I'm not sure what is the suggested change?

Reply:
Only suggestion is to omit the colon ":" after "such that"